nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+          openshift.io/imported-from: sonatype/nexus
+        from:
+          kind: DockerImage
+          name: sonatype/nexus
+        importPolicy: {}
+        name: latest
+        referencePolicy:
+          type: Source
+
+- apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/basic-auth
+  metadata:
+    labels:
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-nexus-credentials
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    username: admin
+    password: ${NEXUS_PASSWORD}
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ${APPLICATION_NAME}-nexus-service
+  data:
+    url: http://nexus-${APPLICATION_NAME}.${DOMAIN_SUFFIX}/nexus/service/local
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      volume.beta.kubernetes.io/storage-class: ${STORAGE_CLASS}
+    name: nexus-pv
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${VOLUME_NEXUS_CAPACITY}
+
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    labels:
+      app: nexus
+    name: nexus
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      app: nexus
+      deploymentconfig: nexus
+    strategy:
+      activeDeadlineSeconds: 21600
+      recreateParams:
+        timeoutSeconds: 600
+      resources: {}
+      type: Recreate
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftNewApp
+        labels:
+          app: nexus
+          deploymentconfig: nexus
+      spec:
+        containers:
+          - image: nexus:latest
+            imagePullPolicy: Always
+            livenessProbe:
+              exec:
+                command:
+                  - echo
+                  - ok
+              failureThreshold: 3
+              initialDelaySeconds: 30
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 1
+            name: nexus
+            ports:
+              - containerPort: 8081
+                protocol: TCP
+            readinessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /nexus
+                port: 8081
+                scheme: HTTP
+              initialDelaySeconds: 30
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 1
+            resources: {}
+            terminationMessagePath: /dev/termination-log
+            terminationMessagePolicy: File
+            volumeMounts:
+              - mountPath: /sonatype-work
+                name: nexus-volume-1
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
```

Comment:
From line 625 as these are default params, they could be remove for clarity

Reply:
ok